Warsaw_Tram/MapViewController.swift

```diff
@@ -13,10 +13,15 @@ import CoreLocation
 class MapViewController: UIViewController, AlertHelperProtocol {
 
     @IBOutlet var map: MKMapView!
+    
     var locationManager = CLLocationManager()
     var userLocationDefined = false
     var userLocation: CLLocationCoordinate2D?
     let tramViewModel = TramViewModel()
+    var allActiveTrams = [Tram]()
```

Comment:
To mant class parameters. Remove all you use only once in whole class and use local var/let inside functions.